code/cranking/retreat-market-nodes/retreat-market-nodes.preview.en.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * This calls zeta's permissionless retreat market nodes function through the Exchange object.
+ * Cranks zeta's on-chain volatility, retreat and interest functionality similiar to update pricing.
+ */
+async function retreatMarketNodes(asset: assets.Asset) {
```

Comment:
this file is not necessary anymore, retreating market nodes is a legacy process that was only relevant for options + dated futures markets